engine/interfaces/src/main/scala/org/trustedanalytics/atk/domain/frame/load/HBaseSource.scala

```diff
@@ -0,0 +1,50 @@
+/*
+// Copyright (c) 2015 Intel Corporation 
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+*/
+
+package org.trustedanalytics.atk.domain.frame.load
+
+import org.apache.commons.lang3.StringUtils
+import org.trustedanalytics.atk.domain.frame.FrameReference
+import org.trustedanalytics.atk.domain.schema.DataTypes.DataType
+import org.trustedanalytics.atk.engine.plugin.ArgDoc
+
+/**
+ * Arguments for the plug-in
+ * @param destination destination frame
+ * @param tableName hbase table name
+ * @param schema hbase schema
+ * @param startTag optional start tag for filtering
+ * @param endTag optional end tag for filtering
+ */
+case class HBaseArgs(
+    @ArgDoc("""DataFrame to load data into.
+Should be either a uri or id.""") destination: FrameReference,
+    @ArgDoc("""table name""") tableName: String,
+    @ArgDoc("""hbase schema""") schema: List[HBaseSchemaArgs],
```

Comment:
Need to explain the schema better here, unfortunately the doc gen won't pick up the docs from HBaseSchemaArgs